server/views/case-summary-record-order.njk

```diff
@@ -90,25 +90,25 @@
     {%- set absentComplied = [] -%}
     {%- set attendedFailed = [] -%}
     {%- set absentFailed = [] -%}
-    {% for description, complied in attendedAppointments.complied.flat(Infinity) | groupby(params.getPath("contactType.description")) %}
```

Comment:
The attendance views wouldn't load with flat(Infinity) applied, I've removed and the integration tests are now passing.